package.json

```diff
@@ -32,8 +32,6 @@
     "build:demo": "tools/build/webcomponent/prepare-wc-pages.sh",
     "demo": "tools/build/webcomponent/prepare-wc-pages.sh --serve",
     "generate-api": "tools/generate-api.sh",
-    "generate-api:gn4": "tools/generate-api.sh gn4",
```

Comment:
I think it was quite useful to have a list of possible APIs to call `generate-api` with, instead of a generic task which needs a folder name; but it doesn't matter much